centreon/www/include/monitoring/objectDetails/serviceDetails.php

```diff
@@ -109,7 +109,7 @@
 }
 
 // Check if host is found
-$host_id = getMyHostID($host_name);
+$host_id = (int) getMyHostID($host_name);
 
 if (!is_null($host_id)) {
```

Comment:
🚨 Something is wrong because the possible return value `NULL` from `getMyHostID` will be cast to `0` and the following `if` will be **always** `TRUE`.